overlord/snapstate/snapmgr_test.go

```diff
@@ -753,20 +752,11 @@ func (s *snapmgrTestSuite) TestUpdateSameRevisionRunThrough(c *C) {
 	s.settle()
 	s.state.Lock()
 
-	expected := []fakeOp{
-		{
-			op:       "download",
-			macaroon: s.user.Macaroon,
-			name:     "some-snap",
-			channel:  "channel-for-7",
-		},
-	}
-
```

Comment:
maybe is just the diff that is hard to read but does this have a matching "s.fakeStore.downloads, DeepEquals" ?